program/rust/src/processor/upd_price.rs

```diff
@@ -144,6 +144,7 @@ pub fn upd_price(
     let mut publisher_index: usize = 0;
     let latest_aggregate_price: PriceInfo;
 
+    #[cfg(feature = "pythnet")]
```

Comment:
Wait why is this only pythnet

Reply:
yeah this should be for the chunk below, good catch